helper/consts/replication.go

```diff
@@ -119,21 +119,12 @@ func (r *ReplicationState) AddState(flag ReplicationState)     { *r |= flag }
 func (r *ReplicationState) ClearState(flag ReplicationState)   { *r &= ^flag }
 func (r *ReplicationState) ToggleState(flag ReplicationState)  { *r ^= flag }
 
-func (r *ReplicationState) IsLeader() bool {
-	// Is Vault clustered and this instance is the leader?
-	if r.HasState(ReplicationPerformancePrimary) {
-		return true
-	}
-	if r.HasState(ReplicationDRPrimary) {
-		return true
-	}
-	// Is replication disabled so Vault is standalone?
-	if r.HasState(ReplicationPerformanceDisabled) {
+func (r *ReplicationState) IsFollower() bool {
+	if r.HasState(ReplicationPerformanceSecondary) {
```

Comment:
For secret/auth backends this is too dependent on if it's a local mount. I don't think this function is needed since it can't contain all the information necessary to answer the question truthfully. I'd just put this logic into the database backend

Reply:
OK! My only concern is that if there were to be more states added later, the logic elsewhere might cease to be correct and it would be difficult to hunt down. 

I will go ahead and update it. Perhaps we should also think about adding more of a Swiss army knife in `ReplicationState` for stuff like this so it's encapsulated and easier to keep correct, but we could do that separately.